app/assets/javascripts/profile_table.js

```diff
@@ -0,0 +1,77 @@
+var ProfileTable = (function(){
+  var widget, selector;
+
+  ProfileTable.prototype = {
+    append: function(){
+      widget = $(selector).editableTableWidget();
+      widget.on('change', parseTableToJSON.bind(this));
+
+      addClickListenersToAddRow();
+      addClickListenersToDeleteRow();
+      parseTableToJSON();
+    }
+  };
+
+  function parseTableToJSON(){
+    $("#testing_ground_technology_profile").text(tableToProfile());
+  };
+
+  function tableToProfile(){
+    var technologyProfile = _.map(tableRows(), function(tableRow){
+      return rowToTechnologyObject(tableRow);
+    });
+    return JSON.stringify(_.groupBy(technologyProfile, 'node'));
```

Comment:
I should add... I have no problem pulling in Underscore provided we're actually going to use it more. :smiley: